From 4a728e4772d53767fcbccd0b2b7e7278c961ce34 Mon Sep 17 00:00:00 2001
From: Andy Green <agreen@pads.home.warmcat.com>
Date: Sat, 23 Aug 2008 11:01:25 +0100
Subject: [PATCH] From: Andrzej Zaborowski <balrogg@gmail.com>

fix-accel-irq-mismatch.patch

I just found a while to start doing something cool with the
accelerometers but I hit #1613 (both accelerometer nodes can't be read
concurrently for longer than a moment). Turns out to be very silly.
I'll continue the cool stuff another day,
Cheers
---
 arch/arm/mach-s3c2440/mach-gta02.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/mach-s3c2440/mach-gta02.c b/arch/arm/mach-s3c2440/mach-gta02.c
index 7d4cc67..eadf88a 100644
--- a/arch/arm/mach-s3c2440/mach-gta02.c
+++ b/arch/arm/mach-s3c2440/mach-gta02.c
@@ -1065,7 +1065,7 @@ void gta02_lis302dl_bitbang_read(struct lis302dl_info *lis)
 	s8 x, y, z;
 #endif
 
-	local_save_flags(flags);
+	local_irq_save(flags);
 
 	/*
 	 * Huh.. "quirk"... CS on this device is not really "CS" like you can
-- 
1.5.6.5

